### ISSUE TITLE
Okteto Up fails to launch the remote  SSH window

### ISSUE THREAD

**Issue by rberrelleza** [okteto/remote-kubernetes#159]
**Describe the bug**
> Hey guys, I am getting ssh error during VSCode remote development setup. When i do Okteto Up, it setup my pod but the new VS code window is not able to get ssh connection
I am using okteto 1.3.4, eks, k8s 1.20, VS Code 1.64

**Logs**

```
14:04:25.805] Log Level: 2
[14:04:25.809] remote-ssh@0.66.0
[14:04:25.809] darwin x64
[14:04:25.836] SSH Resolver called for "ssh-remote+ftu-conv-test.okteto", attempt 1
[14:04:25.837] "remote.SSH.useLocalServer": true
[14:04:25.838] "remote.SSH.path": undefined
[14:04:25.838] "remote.SSH.configFile": undefined
[14:04:25.839] "remote.SSH.useFlock": true
[14:04:25.839] "remote.SSH.lockfilesInTmp": false
[14:04:25.839] "remote.SSH.localServerDownload": auto
[14:04:25.840] "remote.SSH.remoteServerListenOnSocket": false
[14:04:25.840] "remote.SSH.showLoginTerminal": false
[14:04:25.840] "remote.SSH.defaultExtensions": []
[14:04:25.840] "remote.SSH.loglevel": 2
[14:04:25.841] SSH Resolver called for host: ftu-conv-test.okteto
[14:04:25.841] Setting up SSH remote "ftu-conv-test.okteto"
[14:04:25.846] Acquiring local install lock: /var/folders/bf/vgfwfyf16s52mn0py7s30x6h0000gp/T/vscode-remote-ssh-d23af3b0-install.lock
[14:04:25.863] Looking for existing server data file at /Users/dkumar11/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-d23af3b0-b3318bc0524af3d74034b8bb8a64df0ccf35549a-0.66.0/data.json
[14:04:25.864] Using commit id "b3318bc0524af3d74034b8bb8a64df0ccf35549a" and quality "stable" for server
[14:04:25.870] Install and start server if needed
[14:04:25.878] PATH: /Users/dkumar11/.nvm/versions/node/v15.14.0/bin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/usr/local/bin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Library/Apple/usr/bin:/Users/dkumar11/Documents/softwares/apache-maven-3.6.2/bin:/Users/dkumar11/go/bin:/Users/dkumar11/Documents/softwares/gradle-5.5/bin
[14:04:25.878] Checking ssh with "ssh -V"
[14:04:25.923] > OpenSSH_8.8p1, OpenSSL 1.1.1l  24 Aug 2021

[14:04:25.929] askpass server listening on /var/folders/bf/vgfwfyf16s52mn0py7s30x6h0000gp/T/vscode-ssh-askpass-c11aeb902e1d8248b4c31202419b7b676aad76a7.sock
[14:04:25.929] Spawning local server with {"serverId":1,"ipcHandlePath":"/var/folders/bf/vgfwfyf16s52mn0py7s30x6h0000gp/T/vscode-ssh-askpass-b61e897e905db1d49cfda83edfc8aab11e6766ca.sock","sshCommand":"ssh","sshArgs":["-v","-T","-D","62559","-o","ConnectTimeout=15","ftu-conv-test.okteto"],"dataFilePath":"/Users/dkumar11/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-d23af3b0-b3318bc0524af3d74034b8bb8a64df0ccf35549a-0.66.0/data.json"}
[14:04:25.930] Local server env: {"DISPLAY":"1","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"/Users/dkumar11/.vscode/extensions/ms-vscode-remote.remote-ssh-0.66.0/out/local-server/askpass.sh","VSCODE_SSH_ASKPASS_NODE":"/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer)","VSCODE_SSH_ASKPASS_MAIN":"/Users/dkumar11/.vscode/extensions/ms-vscode-remote.remote-ssh-0.66.0/out/askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"/var/folders/bf/vgfwfyf16s52mn0py7s30x6h0000gp/T/vscode-ssh-askpass-c11aeb902e1d8248b4c31202419b7b676aad76a7.sock"}
[14:04:25.932] Spawned 24355
[14:04:26.174] > local-server-1> Spawned ssh, pid=24358
[14:04:26.192] stderr> OpenSSH_8.8p1, OpenSSL 1.1.1l  24 Aug 2021
[14:04:26.465] stderr> Unable to negotiate with ::1 port 22100: no matching host key type found. Their offer: ssh-rsa
[14:04:26.467] > local-server-1> ssh child died, shutting down
[14:04:26.474] Local server exit: 0
[14:04:26.474] Received install output: local-server-1> Spawned ssh, pid=24358
OpenSSH_8.8p1, OpenSSL 1.1.1l  24 Aug 2021
Unable to negotiate with ::1 port 22100: no matching host key type found. Their offer: ssh-rsa
local-server-1> ssh child died, shutting down
```

**Comment by rberrelleza** [comment]
https://github.com/okteto/okteto/issues/1959 is the root cause of this issue. As a workaround, you can add the following configuration to $HOME/.ssh/config to enable support for SSH-RSA keys:

```
Host *
  HostKeyAlgorithms +ssh-rsa  
```

**Comment by deepk2u** [comment]
@rberrelleza This might break some of the existing ssh connections, I think if you do `*.okteto`, that will also help. I tested it and it works.

```
Host *.okteto
  HostKeyAlgorithms +ssh-rsa  
  ```

**Comment by rberrelleza** [comment]
This is fixed in the 0.4.1 release
